test/realtime/tenants/authorization_test.exs

```diff
@@ -0,0 +1,145 @@
+defmodule Realtime.Tenants.AuthorizationTest do
+  require Phoenix.ChannelTest
+  use RealtimeWeb.ConnCase
```

Comment:
For what is worth, I like to explicitly add `async: true | false` to all of my tests and explicitly leave a comment when it has to be `async: false` with the reasons why.